runtime/cpp/instance_segmentation/yolov5seg/README.md

```diff
@@ -1,38 +1,73 @@
-**Last HailoRT version checked - 4.17.0**
+Detection and Instance Segmentation example
+===========================================
 
-**Disclaimer:** <br />
-This code example is provided by Hailo solely on an “AS IS” basis and “with all faults”. No responsibility or liability is accepted or shall be imposed upon Hailo regarding the accuracy, merchantability, completeness or suitability of the code example. Hailo shall not have any liability or responsibility for errors or omissions in, or any business decisions made by you in reliance on this code example or any part of it. If an error occurs when running this example, please open a ticket in the "Issues" tab.<br />
-Please note that this example was tested on specific versions and we can only guarantee the expected results using the exact version mentioned above on the exact environment. The example might work for other versions, other environment or other HEF file, but there is no guarantee that it will.
+This script performs object detection and instance segmentation on a video using Hailo.
+It annotates detected objects with bounding boxes and draws a mask on each instance in every frame of the video.
 
+Requirements
+------------
 
-This is a HailoRT C++ API yolov5seg detection + instance segmentation example.
+- hailo_platform==4.17.0
+- OpenCV (cv2)
+- g++-9
+ 
 
-The example does the following:
+Usage
+-----
 
-Creates a device (pcie)
-Reads the network configuration from a yolov5eg HEF file
-Prepares the application for inference
-Runs inference and postprocess on a given video file
-Draws the detection boxes on the original video
-Colors the detected objects pixels
-Prints the object detected + confidence to the screen
-Prints statistics
-NOTE: Currently supports only devices connected on a PCIe link.
-
-1. Dependencies:
-    - OpenCV, and g++-9:
-    ``` bash
+1. Install the requierments:
+    ```shell script
     sudo apt-get install -y libopencv-dev gcc-9 g++-9
     ```
-2. Build the project build.sh
-3. Run the executable:
-    ``` bash
-    ./build/segmentation_example_cpp -hef=HEF_PATH -path=VIDEO_PATH
+
+2. Install PyHailoRT
+    - Download the HailoRT whl from the Hailo website - make sure to select the correct Python version. 
+    - Install whl:
+        ```shell script
+        pip install
+        ```
+
+3. Download example files:
+    ```shell script
+    ./download_files.sh
+    ```
+
+5. Run the script:
```

Comment:
all of the steps besides #1 are not relevant

Reply:
I updated this part in the second commit